### DISCUSSION TITLE
QUESTION: PWM Version to Eurorack

### DISCUSSION THREAD

**Discussion by bmmcwhirt** [frntc/SIDKick-pico#86] [Q&A]
With the PWV version is it possible to convert the output to something that can be input into a EuroRack 1V/Octive input?

**Comment by frntc** [comment]
I'm not into synthesizers and don't know how they take the input, but I tend to say that it won't work, because the PWM output normally needs an additional circuit (which is not on the SKpico PCB, because the C64 mainboard has circuit which is a working substitute).
